src/v0/sources/shopify/config.js

```diff
@@ -102,7 +102,8 @@ const SUPPORTED_TRACK_EVENTS = [
   'orders_paid',
   'orders_partially_fullfilled',
 ];
-
+const timeDifferenceForCartEvents = 10000; // in micro seconds as we will be compairing it in timestamp 
+const useRedisDatabase = process.env.USE_REDIS_DB || true;
```

Comment:
Probably we need to change into below way
```suggestion
const useRedisDatabase = process.env.USE_REDIS_DB === 'true' || true;
```

Reply:
Done